tests/integration/config/plugins/integration-tests/data/tests/tests_spawn_limits.txt

```diff
@@ -0,0 +1,254 @@
+test-data "Spawn Limits Test"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system "Terra Incognita"
+		planet Ruin
+		clearance
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 70
+				bunks 3
+				"cargo space" 50
+				drag 2.1
+				"engine capacity" 40
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 130
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 20
+				"thrust" 100
+				"reverse thrust" 100
+				"turn" 1000
+				"energy generation" 10
+			outfits
+				Hyperdrive
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			position 0 0
+			engine -9 38 1
+			engine 9 38 1
+			system "Terra Incognita"
+			planet Ruin
+		account
+			credits 100000000
+			score 400
+			history
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		conditions
+			"Ruin: Landing: offered"
+		event
+			date 1 1 3012
+			system "Terra Incognita"
+				fleet "Non-Disabled A Test Fleet"
+					category test
+					period 1
+					non-disabled limit 1
+					ignore enemy strength
+				fleet "Disabled B Test Fleet"
+					category test
+					period 1
+					limit 3
+					ignore enemy strength
+				fleet "Disabled C Test Fleet"
+					category test
+					period 1
+					limit 0
+					initial count 1
+					ignore enemy strength
+				fleet "Disabled 2 Test Fleet"
+					period 100000000
+					limit 0
+					initial count 1
+
+
+
+fleet "Non-Disabled A Test Fleet"
+	government "Test Government 1 A"
+	cargo 0
+	personality staying heroic
+	variant 60
+		"Aphid"
+
+fleet "Disabled B Test Fleet"
+	government "Test Government 1 B"
+	cargo 0
+	personality staying derelict heroic
+	variant 60
+		"Star Barge"
+
+fleet "Disabled C Test Fleet"
+	government "Test Government 1 C"
+	cargo 0
+	personality staying derelict heroic
+	variant 60
+		"Shuttle"
+
+government "Test Government 1 A"
```

Comment:
We dont need all of these governments, that said since this is an integration test it doesnt need to be smart

Reply:
This is the only way to detect which ship is which in the boarding missions.